### PR TITLE
Updated the Japanese expression and added some translations.

### PR DESCRIPTION
**Pull request by nyatla**
Fixed Japanese phrases.
Add some translations.

As a known problem, there is a problem with the display width of the network selection screen.

### CODE REVIEW

File: app/_locales/ja/messages.json [hunk 1]
```diff
@@ -161,13 +173,13 @@
...
-    "message": "別のアカウントからデポジット"
+    "message": "別のアカウントから入金"
   },
   "depositShapeShift": {
-    "message": "ShapeShiftでデポジット"
+    "message": "ShapeShiftで入金"
   },
   "depositShapeShiftExplainer": {
-    "message": "あなたが他の暗号通貨を持っているなら、Etherにトレードしてダイレクトにメタマスクウォレットへのデポジットが可能です。アカウント作成は不要。"
+    "message": "他の暗号通貨をEtherと交換してMetaMaskのウォレットへ入金できます。アカウント作成は不要です。"
```
Comment (kumavis) [inline app/_locales/ja/messages.json]: 暗号通貨 or 仮想通貨?

**Review by kumavis — APPROVED** [review]

**Comment by kumavis** [comment]
thanks @nyatla

**Comment by kumavis** [comment]
@nyatla Please join our translator team 🦊 
https://github.com/MetaMask/

File: app/_locales/ja/messages.json (hunk 1 above)
Comment (nyatla) [reply app/_locales/ja/messages.json]: The 暗号通貨 is a term used by Japanese who is familiar with CryptoCurrency. In general it is a 仮想通貨. I think that people using Ethereum prefer 暗号通貨.

**Pull request merged**